src/wallet/api/wallet2_api.h

```diff
@@ -304,6 +368,7 @@ struct SubaddressAccountRow {
     std::string m_balance;
     std::string m_unlockedBalance;
 public:
+    // QUESTION : afaik this is unused, can we remove it?
```

Comment:
I would vote for removing. That whole `tx_extra` mechanism is very controversial, problematic privacy-wise, and we once were on the verge of deleting it altogether. We sure don't want to give people ideas to use it.

Where did you check for use?

Reply:
I thought this was some other sort of "extra" apart from `tx_extra`. Why would you store `tx_extra` inside  `SubaddressAccountRow`?

> Where did you check for use?

Did the following grep searches:
```grep -r "\->extra" src/```
```grep -r "\.extra" src/```
Then looked through the noise and can say with relatively high certainty that none of the results involved the `SubaddressAccountRow` `extra`. Also just did a quick search in github for the GUI for the same strings and found nothing, but I did not check if it's used by any third party.

Anyways, I'd still vote "remove".

Reply:
Oh, of course, my bad, nothing to do with `tx_extra`.

It was introduced 8 years ago with the initial subaddresses PR. I am pretty sure that it's not only unused by any client, it's never set!

So, yes, I agree with remove.